chef_master/source/perform_compliance_scan.rst

```diff
@@ -11,102 +11,40 @@ Perform a Compliance Scan in Chef Automate
-
-   .. code-block:: ruby
-
-      source 'https://supermarket.chef.io'
-
-      metadata
-
-      cookbook 'audit'
-
-   You could also pull the audit cookbook from GitHub, but make sure that the version you use is 4.0 or later.
-
-   .. code-block:: ruby
-
-      source 'https://supermarket.chef.io'
-
-      metadata
-
-      cookbook 'audit', '~> 4.0', github: 'chef-cookbooks/audit'
-
-   A ``Berksfile.lock`` file is generated for you referencing the audit cookbook and any dependencies.
-
-#. Run ``berks vendor`` from the same directory as your ``Berksfile`` to pull down the audit cookbook and its dependencies to your local workstation. If a directory name is not specified, then they will be copied into the ``berks-cookbooks`` directory. And depending on your workstation, you may have to use ``sudo`` when calling berks commands.
-
-   .. code-block:: bash
-
-      $ sudo berks vendor
-
-        Resolving cookbook dependencies...
-        Fetching 'audit' from https://github.com/chef-cookbooks/audit.git (at master)
-        Fetching 'simple_cookbook' from source at .
-        Fetching cookbook index from https://supermarket.chef.io...
-        Using audit (4.0.0) from https://github.com/chef-cookbooks/audit.git (at master)
-        Using simple_cookbook (0.1.0) from source at .
-        Using compat_resource (12.19.0)
-        Vendoring audit (4.0.0) to /Users/dwrede/Repos/audit_test/cookbooks/simple_cookbook/berks-cookbooks/audit
-        Vendoring compat_resource (12.19.0) to /Users/dwrede/Repos/audit_test/cookbooks/simple_cookbook/berks-cookbooks/compat_resource
-        Vendoring simple_cookbook (0.1.0) to /Users/dwrede/Repos/audit_test/cookbooks/simple_cookbook/berks-cookbooks/simple_cookbook
-
-#. Run ``knife cookbook upload`` to upload the audit cookbook and its dependent cookbooks. To upload more than one cookbook at a time, use the ``-a`` and ``-o`` options to specify all cookbooks from a given directory.
-
-   .. code-block:: bash
-
-      $ knife cookbook upload -a -o berks-cookbooks
-
-        Uploading audit        [4.0.0]
-        Uploading compat_resource [12.19.0]
-        Uploading simple_cookbook [0.1.0]
-        Uploaded all cookbooks.
-
 Configure Data Collection on Chef server
 -------------------------------------------------------
 
-To proxy node data through Chef server to Chef Automate, you must update the ``/etc/opscode/chef-server.rb`` file on your Chef server. This is needed for convergence status and general node data, but it is also true for proxying audit run data from nodes back to Chef Automate.
+To send node data through Chef server to Chef Automate, you must update the ``/etc/opscode/chef-server.rb`` file on your Chef server. This is needed for convergence status and general node data, but it is also true for sending audit run data from nodes back to Chef Automate.
```

Comment:
`convergence` or `converge` status? I prefer the latter.

Reply:
OK.